src/core_plugins/kibana/public/context/app.html

```diff
@@ -0,0 +1,142 @@
+<div class="kuiLocalNav">
+  <div class="kuiLocalNavRow">
+    <div class="kuiLocalNavRow__section">
+      <div class="kuiLocalTitle">
+        Surrounding Entries in {{ contextApp.state.queryParameters.indexPattern.id }}
+      </div>
+    </div>
+  </div>
+  <div class="kuiLocalNavRow kuiLocalNavRow--secondary">
+    <div class="kuiLocalTabs">
+      <div class="kuiLocalTab kuiLocalTab-isSelected" ng-bind="contextApp.state.queryParameters.anchorUid"></div>
+    </div>
+  </div>
+</div>
+
+<!-- Error feedback -->
+<div
+  class="kuiViewContent kuiViewContentItem"
+  ng-if="contextApp.state.loadingStatus.anchor === contextApp.constants.LOADING_STATUS.FAILED"
+>
+  <div class="kuiInfoPanel kuiInfoPanel--error kuiVerticalRhythm">
+    <div class="kuiInfoPanelHeader">
+      <span class="kuiInfoPanelHeader__icon kuiIcon kuiIcon--error fa-warning"></span>
+      <span class="kuiInfoPanelHeader__title">
+        Problem with query
+      </span>
+    </div>
+
+    <div class="kuiInfoPanelBody">
+      <div class="kuiInfoPanelBody__message">
+        Failed to load the anchor document. Please reload or visit
+        <a ng-href="{{ contextApp.state.navigation.discover.url }}">Discover</a>
+        to select a valid anchor document.
+      </div>
+    </div>
+  </div>
+</div>
+
+<div
+  class="kuiViewContent kuiViewContentItem"
+  ng-if="contextApp.state.loadingStatus.anchor !== contextApp.constants.LOADING_STATUS.FAILED"
+  role="main"
+>
+  <!-- Controls -->
+  <div class="kuiBar kuiVerticalRhythm">
+    <div class="kuiBarSection">
+      <context-size-picker
+        count="contextApp.state.queryParameters.predecessorCount"
+        data-test-subj="predecessorCountPicker"
+        is-disabled="contextApp.state.loadingStatus.anchor !== contextApp.constants.LOADING_STATUS.LOADED"
+        on-change-count="contextApp.actions.fetchGivenPredecessorRows"
+      ></context-size-picker>
+      <span>newer entries</span>
+      <context-loading-button
+        data-test-subj="predecessorLoadMoreButton"
+        is-disabled="![
+          contextApp.constants.LOADING_STATUS.LOADED,
+          contextApp.constants.LOADING_STATUS.FAILED,
+        ].includes(contextApp.state.loadingStatus.predecessors)"
+        icon="'fa-chevron-up'"
+        ng-click="contextApp.actions.fetchMorePredecessorRows()"
+      >
+        Load more
+      </context-loading-button>
+      <span
+        class="kuiStatusText kuiStatusText--warning"
+        ng-if="(contextApp.state.loadingStatus.predecessors === contextApp.constants.LOADING_STATUS.LOADED)
+          && (contextApp.state.queryParameters.predecessorCount > contextApp.state.rows.predecessors.length)"
+      >
+        <span class="kuiStatusText__icon kuiIcon fa-bolt"></span>
+        <span ng-bind-template="Only {{ contextApp.state.rows.predecessors.length }} entries newer than the anchor could be found."></span>
+      </span>
+    </div>
```

Comment:
What do you think of putting the button first, and changing its text so it's clear that it's an incremental action?

![image](https://cloud.githubusercontent.com/assets/1238659/23001418/fc48b30a-f399-11e6-8f93-2e1b2fc99727.png)